### PR TITLE
Use consistent normalization for Zernikes

### LINKED ISSUE #1002: Consistent normalization of Legendre and Zernike polys
Currently, users have to normalize by (2n + 1) / 2 for Legendre coefficients and similarly by 1 / pi for Zernikes. If we're going to do this, then I think we should also remove the Zernike code that multiplies the Zernike polynomials by √((2n + 2) / epsilon). We should decide: either the user has to apply the integration factors consistently or they don't.

### PR DESCRIPTION
**Pull request by lindsayad**
This leaves it to the users to handle all normalization factors of Zernike polynomials. This is consistent with the lack of normalization of the spatial Legendre polynomials.

Closes #1002

### CODE REVIEW

**Comment by paulromano** [comment]
Thanks @lindsayad! You should also update the description of `ZernikeFilter` in `openmc/filter_expansion.py` (namely, on the equations for the Zernike polynomials, remove the normalization factor with n).

**Comment by lindsayad** [comment]
@paulromano Could you share the Noll paper with me? INL doesn't have access to it

**Comment by paulromano** [comment]
@lindsayad Just sent it to you via email

File: openmc/filter_expansion.py [hunk 1]
```diff
@@ -335,7 +335,8 @@ class ZernikeFilter(ExpansionFilter):
         \frac{n+m}{2} - k)! (\frac{n-m}{2} - k)!} \rho^{n-2k}.
 
     With this definition, the integral of :math:`(Z_n^m)^2` over the unit disk
-    is exactly :math:`\pi` for each polynomial.
+    is :math:`\frac{\epsilon_m\pi}{2n+2}` for each polynomial where :math:`\epsilon_m` is
+    2 if math:`m` equals 0 and 1 otherwise.
```
Comment (paulromano) [inline openmc/filter_expansion.py]: missing a colon before `math:`

**Comment by paulromano** [comment]
Thanks again!

**Pull request merged**